components/tensorboard-controller/.gitignore

```diff
@@ -0,0 +1,24 @@
+
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+bin
+
```

Comment:
Should we add the `tensorboard-controller/manager` binary here?

Reply:
added. btw what's this for ? (I asked before without it my local run works as well).

Reply:
The binary still seems to be checked in, do we want to check this in?
https://github.com/kubeflow/kubeflow/blob/d97f7b272e52078e802807cdba760e4d0c98daab/components/tensorboard-controller/manager

Reply:
remove, thanks !